tests/remotes/__init__.py

```diff
@@ -40,7 +40,7 @@ def docker():
     import os
 
     # See https://travis-ci.community/t/docker-linux-containers-on-windows/301
-    if os.environ.get("TRAVIS") and os.name == "nt":
+    if os.environ.get("GITHUB_ACTION") and os.name == "nt":
```

Comment:
@pared, use `CI`, this way it will be CI/CD-agnostic and easier to migrate in future.
```suggestion
    if os.environ.get("CI") and os.name == "nt":
```

Reply:
Good idea, thoguh I wonder whether we should have any check besides "nt". Docker itself does not allow pulling linux images on windows, so its not like temporary check.

Reply:
Interestingly, windows doc's seems to tell different story than tests log:
https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/linux-containers#:~:text=Docker%20has%20been%20able%20to,machine%20running%20on%20Hyper%2DV.
maybe there needs to be some configuration done to be able to run `docker` fixture on our CI pipeline.